README.md

```diff
@@ -46,6 +46,8 @@ Flags:
       --accounts                            Export per account metrics
       --gatewayz                            Export gateway metrics
       --jsz string                          Export jsz metrics optionally, one of: all|streams|consumers
+      --jsz-leaders-only bool               Fetch jsz data from stream and consumer leaders only
+      --jsz-filter stringArray              Export jsz consumer metrics that match selected filters
```

Comment:
So the users who types `./nats-surveyor -h` will see this:
```
     --jsz-filter stringArray              Fetch selected list of jsz metrics
```

Will still wonder what the possible values for the string array are.
And will need to go to README.md on the website to find out, thus making help an incomplete solution.
This UX can be improved by using an enum here(see comment) in the previous commit.

I understand that you did not request my review, but IMHO, we should not land with this UI.

Reply:
so you want the list of values here? can you put the text that should in the readme and we copy paste it there?

Reply:
also feel free to push to the branch too in how it should look like, thanks for taking a look